src/components/StoreMigrator/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react'
+import { MIGRATION_ADDRESS } from 'src/routes/routes'
+import { saveToStorage, loadFromStorage } from 'src/utils/storage'
+
+const MIGRATION_PATH_BASE_URL = 'http://localhost:3001/#'
+const MIGRATION_PATH = `${MIGRATION_PATH_BASE_URL}${MIGRATION_ADDRESS}`
+const APPS_STORAGE_MIGRATION_DONE = 'APPS_STORAGE_MIGRATION_DONE'
+
+const StoreMigrator: React.FC = () => {
+  const [migrationDone, setMigrationDone] = useState(false)
+  const [checkMigrationDone, setCheckMigrationDone] = useState(false)
+  const [loaded, setLoaded] = useState(false)
+
+  // Redirect if migration already happened
```

Comment:
Why do you need to redirect? Isn't this route supposed to be opened in an iframe? We don't want to open any other URL in the iframe.